build/bin/generate.php

```diff
@@ -513,3 +513,19 @@ function generate(Generator $generator, array $options)
         'ArrayOfLong',
     ],
 ] + $defaultOptions);
+
+generate($generator, [
+    'inputFile' => 'https://api.direct.yandex.com/v5/businesses?wsdl',
```

Comment:
```suggestion
    'inputFile' => 'https://api.direct.yandex.com/v501/businesses?wsdl',
```

Чтобы не было конфликтов с другими сервисами. Так как в wsdl подключаются другие общие схемы и там типы могут отличаться. И результат генерации контрактов, как следствие.

Reply:
@yethee поправил  на https://api.direct.yandex.com/v501/businesses?wsdl , результат генерации файлов вроде как тот же что и прошлый раз , думаю что это не критично или же я чтото сделал не так ?

Reply:
Значит сейчас схемы для `v5` и `v501` идентичны. Летом различались, когда происходил процесс миграции на ЕПК.

Думаю, лучше оставить `v501` для согласованности и чтобы минимизировать появление конфликтов в будущем. Не известно как будут развиваться схемы для этих версий.